Packs/Druva/Integrations/Druva/Druva_test.py

```diff
@@ -1,3 +1,75 @@
+import pytest
+import json
+
+def test_test_module(requests_mock):
+    from Druva import Client,test_module
+    requests_mock.get(f"https://apis.druva.com/realize/ransomwarerecovery/v1/quarantineranges", json=[], status_code=200)
+    client = Client(
+        base_url='https://apis.druva.com/',
+        verify=False,
+        headers={
+            'Authentication': 'Bearer some_api_key'
+        }
+    )
+    assert test_module(client) == 'ok'
+def test_test_module_Command_Invalid_input(requests_mock):
+    from Druva import Client, test_module
+    requests_mock.get(f"https://apis.druva.com/realize/ransomwarerecovery/v1/quarantineranges",status_code=100)
+    client = Client(
+        base_url='https://apis.druva.com/',
+        verify=False,
+        headers={
+            'Authentication': 'Bearer some_api_key'
+        }
+    )
+    with pytest.raises(RuntimeError) as ve:
+        test_module(client)
+    assert str(ve.value)=='Internal Error'
+
+def test_Execute_command(requests_mock):
+    from Druva import Client,Execute_command
+    client = Client(
+        base_url='https://apis.druva.com/',
+        verify=False,
+        headers={
+            'Authentication': 'Bearer some_api_key'
+        }
+    )
+    
+    with open("test_data/test-data.json") as file:
+        data=json.load(file)
+        commands=["test-module","druva-list-quarantine-ranges","druva-find-device","druva-find-user","druva-find-userDevice","druva-find-sharePointSites","druva-find-sharedDrives",
+                  "druva-quarantine-resource","druva-delete-quarantine-range","druva-view-quarantine-range","druva-update-quarantine-range","druva-list-quarantine-snapshots","druva-delete-quarantined-snapshot",
+                  "druva-endpoint-check-restore-status","druva-endpoint-search-file-hash"]
+        for cur_command in commands:
```

Comment:
(this note is not related to the suggestion above)
your current approach is totally ok, but FYI: you can avoid defining the `commands` list and iterating it by doing something like:
```suggestion
        for command_name, command_data in data.items():
            mocker.patch.object(demisto, "command", return_value=command_name)
            if get_request_url := command_data["get-request-url"]:
                mock_response = command_data["get-response-mock-data"]
                requests_mock.get(get_request_url, status_code=200, json=mock_response)
            if post_request_url := command_data["post-request-url"]:
                mock_response = command_data["post-response-mock-data"]
                requests_mock.post(post_request_url, status_code=200, json=mock_response)
            if put_request_url := command_data["put-request-url"]:
                mock_response = command_data["put-response-mock-data"]
                requests_mock.put(put_request_url, status_code=200, json=mock_response)
            if delete_request_url := command_data["delete-request-url"]:
                mock_response = command_data["delete-response-mock-data"]
                requests_mock.delete(delete_request_url, status_code=200, json=mock_response)
            main()
```
of course, this requires some additional changes in the test-data.json file. As I mentioned above, this change is not required, only my recommendation :)